packages/pds-ember/app/styles/pds/theme/index.scss

```diff
@@ -1,4 +1,199 @@
-// Default Theme (`@use "pds/theme";`)
-// Alternative themes should be consumed via `@use "pds/theme/{name}";`,
-// which would correspond with individual files in this module directory.
-@forward "light";
+$_module: "PDS.Theme";
+@use "../tokens/color";
+@use "../tokens/font";
+@use "../tokens/resolutions";
+@use "../tokens/scale";
+@use "../utils/prefers" as Prefers;
+@use "__shadow" as Shadow;
+// Themes
+@use "dark" as Dark; // EXPERIMENTAL!!!
+@use "light" as Light;
+
+// FIXME: "magic number" outside of defined scale
+// DO NOT use this variable outside of this module!
+//
+// This private var is used to apply a 14px font size to theme variables
+// (i.e., variables defined in THIS file) that require it as part of design
+// specifications.  Because it is a value that doesn't exist in the modular
+// scale (see tokens/scale.scss), we need to keep it centralized to prevent
+// accidental use by consumers and to provide an upgrade path if designs are
+// modified in the future.
+$_size14: 0.875rem; // ~14px
+
+///=========================================================================///
+/// DYNAMIC MEMBERS
+///=========================================================================///
+$backgroundColor: var(--pds-backgroundColor);
+$color: var(--pds-color);
+$focus-color: var(--pds-focus-color);
+// Typography
+$letterSpacing: var(--pds-letterSpacing);
+$lineHeight: var(--pds-lineHeight);
+$lineHeight--dense: var(--pds-lineHeight--dense);
+$paragraphSpacing: var(--pds-paragraphSpacing);
+$wordSpacing: var(--pds-wordSpacing);
+// Action
+$action-color: var(--pds-action-color);
+$action-color--amp1: var(--pds-action-color--amp1);
+$action-color--dim1: var(--pds-action-color--dim1);
+$action-color--dim2: var(--pds-action-color--dim2);
+// Alert
+$alert-color: var(--pds-alert-color);
+$alert-color--amp1: var(--pds-alert-color--amp1);
+$alert-color--dim1: var(--pds-alert-color--dim1);
+$alert-color--dim2: var(--pds-alert-color--dim2);
+// Error
+$error-color: var(--pds-error-color);
+$error-color--amp1: var(--pds-error-color--amp1);
+$error-color--dim1: var(--pds-error-color--dim1);
+$error-color--dim2: var(--pds-error-color--dim2);
+// Info
+$info-color: var(--pds-info-color);
+$info-color--amp1: var(--pds-info-color--amp1);
+$info-color--dim1: var(--pds-info-color--dim1);
+$info-color--dim2: var(--pds-info-color--dim2);
+// Success
+$success-color: var(--pds-success-color);
+$success-color--amp1: var(--pds-success-color--amp1);
+$success-color--dim1: var(--pds-success-color--dim1);
+$success-color--dim2: var(--pds-success-color--dim2);
+// Warning
+$warning-color: var(--pds-warning-color);
+$warning-color--amp1: var(--pds-warning-color--amp1);
+$warning-color--dim1: var(--pds-warning-color--dim1);
+$warning-color--dim2: var(--pds-warning-color--dim2);
+// Borders
+$borderColor: var(--pds-borderColor);
+$borderColor--dim1: var(--pds-borderColor--dim1);
+$borderRadius: var(--pds-borderRadius);
```

Comment:
Notice how each Sass variable resolves to a `var(--pds-*)` value?  This ensures that any usage of Sass variables will automatically wire up to the theming framework.  All of the CSS custom properties are defined in the individual theme files (e.g., `pds/theme/_light`, `pds/theme/_dark`, etc.).